domains/aadarsh.json

```
@@ -0,0 +1,8 @@
+{
+  "owner": {
```

```suggestion
  "owner": {
    "username": "Aadarsh0307",
```

you must add your username